conf/openlibrary-docker.yml

```diff
@@ -0,0 +1,208 @@
+coverstore_config_file: coverstore.yml
+
+loanstatus_url: http://localhost/
+ia_access_secret: foobar
+
+#dev_instance: True
+dev_instance_url: http://127.0.0.1:8080/
+
+memcache_servers:
+    - localhost:11211
+
+plugin_modules:
+    - openlibrary
+    - infogami.plugins.links
+    - infogami.plugins.api
+
+plugin_worksearch:
+    solr: solr:8080
+    spellcheck_count: 3
+    ebook_count_db_parameters:
+        db: openlibrary_ebook_count
+        host: localhost
+
+ia_ol_xauth_s3:
+    s3_key: XXX
+    s3_secret: XXX
+
+ia_loan_api_url: http://127.0.0.1/internal/fake/loans
+ia_availability_api_url: http://127.0.0.1/internal/fake/availability
+ia_xauth_api_url: http://127.0.0.1/internal/fake/xauth
+http_request_timeout: 10
+
+stats_solr: solr:8080
+
+features:
+    upstream: enabled
+    cache_most_recent: enabled
+    recentchanges_v2: enabled
+    history_v2: admin
+    merge-authors:
+        filter: usergroup
+        usergroup: /usergroup/librarians
+    merge-editions: admin
+    undo: enabled
+    dev: enabled
+    lists: enabled
+    lending_v2: enabled
+    stats: enabled
+    stats-header: enabled
+    inlibrary: enabled
+    support: admin
+    dev: enabled
+    superfast: enabled
+    publishers: enabled
+    languages: enabled
+
+upstream_to_www_migration: true
+default_template_root: /upstream
+css_root: /upstream/css
+js_root: /upstream/js
+use_google_cdn: false
+logging_config_file: conf/logging.ini
+email_config_file: conf/email.ini
+
+lists:
+    seeds_db: http://127.0.0.1:5984/seeds
+    editions_db: http://127.0.0.1:5984/editions
+    works_db: http://127.0.0.1:5984/works
+    editions_view: http://127.0.0.1:5984/editions/_fti/_design/seeds/by_seed
+
+admin:
+    olsystem_root: /home/noufal/projects/OL/olsystem/
+    nagios_url: http://monitor.us.archive.org/cgi-bin/nagios3/status.cgi?hostgroup=24.openlibrary&style=detail
+    statsd_server: localhost:9090
+    admin_db: http://127.0.0.1:5984/admin
+
+
+stats: # This section is used to state what stats need to be gathered.
+    pageload.all:
+      filter: all
+      time: total
+
+    pageload.books:
+      filter: url
+      pattern: /books/OL\d+M
+      time: total
+
+    pageload.works:
+      filter: url
+      pattern: /works/OL\d+M
+      time: total
+
+    pageload.authors:
+      filter: url
+      pattern: /authors/OL\d+M
+      time: total
+
+    pageload.author:
+      filter: url
+      pattern: /authors/OL\d+M
+      time: total
+
+    pageload.home:
+      filter: url
+      pattern: ^/$
+      time: total
+
+    pageload.recentchanges:
+      filter: url
+      pattern: /recentchanges
+      time: total
+
+    pageload.lists:
+      filter: url
+      pattern: /lists
+      time: total
+
+    ol.pageload.readapi:
+      filter: url
+      pattern: /api/volumes/.*
+      time: total
+
+
+libraries_admin_email: admin@dev-instance
+
+graphite_base_url: http://ol-admin.us.archive.org:7080
+geoip_database: var/cache/GeoLiteCity.dat
```

Comment:
@mekarpeles what is this for, I don't see it in olsystem?

Reply:
Used in https://github.com/internetarchive/openlibrary/blob/75452cf6f0e3c23e1030395c93e232686ebaa417/openlibrary/core/geo_ip.py